### ISSUE TITLE
Counters can be resest

### ISSUE THREAD

**Issue by gokulwm** [gokulwm/lab-agile-planning#4]
**As a** System Administrator
 **I need** the ability to reset any counters
 **So that** i can start counting the counter from its initial value
   
 ### Details and Assumptions any details helpful for the developers to build the story
 * A reset button is placed near update button
   
 ### Acceptance Criteria  
   
 ```gherkin
 Given i have my counter value as 3
 When i reset it using the button
 Then my counter value should be equal to 0 on retrieving and viewing it
 ```
